django_mongodb/compiler.py

```diff
@@ -86,6 +107,16 @@ def _make_result(self, entity, columns):
             result.append(obj.get(name))
         return result
 
+    def cursor_iter(self, cursor, chunk_size, columns):
+        """Yield chunks of results from cursor."""
+        chunk = []
+        for row in cursor:
+            chunk.append(self._make_result(row, columns))
+            if len(chunk) == chunk_size:
+                yield chunk
+                chunk = []
```

Comment:
So "Chunk" here is the number of items in a list and not bounded by bytes? (confirming as that was the expectation)

Reply:
Right. Django's default for `chunk_size` is 100 (rows), or 2000 if using `QuerySet.iterator()`.